### ISSUE TITLE
Brain Stem Substructures Module

### ISSUE THREAD

**Issue by fmcalcagno** [Deep-MI/FastSurfer#270]
## Feature Description
We would like to train and create a new sub-module to obtain from a T1 weighted image all 4 substructures of the Brain Stem (Pons, MidBrain, Medule, SCP)
...

## Motivation
The volume of the Pons can be very important to understand the state of certain degenerative diseases. 
We are using this module [https://surfer.nmr.mgh.harvard.edu/fswiki/BrainstemSubstructures](url)  from FreeSurfer to calculate the volumes of each region. We believe that FastSurfer could include a module with this substructures and deliver results Faster.

We propose to generate a Dataset including this regions using the same Open Source Datasets that where used to train FastSurfer, and generate a dataset using the FreeSurfer Module. 
The next step would be to create a new Deep Learning model to detect all these 4 different substructures using FastSurferCNN and deliver the module to the community.

We're open to discusion and to new ideas.

**Comment by m-reuter** [comment]
Hi @fmcalcagno , we already prepare a dataset with manually segmented/cleaned brainstem labels and plan to add it later to the upcoming cerebellum sub-segmentation module.

**Comment by fmcalcagno** [comment]
Hi @m-reuter , thank you for you quick response.
Is the new upcoming cerebellum sub-segmentation module including an output (both volumetric and in nifti files) for the four regions of the brain stem (Pons, MidBrain, Medule, SCP) ?

**Comment by m-reuter** [comment]
The upcoming module will only include the cerebellum (see the cerebnet paper). We are working on an extension to also include the Brainstem regions. Until then you need to use FreeSurfer module for it.

**Comment by fmcalcagno** [comment]
We propose to help you with the training of the FastSurferCNN Brain Stem module, we have the hardware and the time if needed.

**Comment by DimToum** [comment]
Hi! Any update on that? (i.e. segmentation module for the subregions of the brain stem)

**Comment by m-reuter** [comment]
No, currently no updates. Still waiting for the manual labels (the usual bottleneck). In the meantime, you should be able to run FreeSurfer's brain stem module on top of FastSurfer.

**Comment by DimToum** [comment]
Ok, thank you. Please let me know if I can help with the manual labeling, I am an academic neuroradiologist.
